### PR TITLE
Upload my module

### PR DESCRIPTION
**Pull request by reyes-simonejake**
Please check my module

### CODE REVIEW

**Review by delumen-ivan — APPROVED** [review]
nice work!

File: errawrs/reyes.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            elif choice == "4":
+                self.summarize_attendance()
+            elif choice == "5":
+                self.reset_attendance()
+            elif choice == "6":
+                print("Exiting the system. Goodbye!")
+                break
+            else:
+                print("Invalid choice. Please try again.")
+
```
Comment (victorioso-daniel-2) [inline errawrs/reyes.py:85]: unnecessary space

**Review by victorioso-daniel — CHANGES_REQUESTED** [review]
module not appearing in the main.py

**Review by mosquito-michaelangelo — CHANGES_REQUESTED** [review]
no calling in main.py

**Review by delumen-ivan — APPROVED** [review]

File: errawrs/reyes.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    # Method 5: 
+    def reset_attendance(self):
+        for name_person in self.records:
+            self.records[name_person] = []
+            self.present_count[name_person] = 0
+        self.total_days = 0
+        print("Attendance records have been reset.\n")
+
+    def menu(self):
```
Comment (reyes-simonejake) [inline errawrs/reyes.py]: ok

**Review by reyes-simonejake — COMMENTED** [review]
Thank you

**Review by victorioso-daniel — APPROVED** [review]
LGTM

**Review by delumen-ivan — APPROVED** [review]

**Review by mosquito-michaelangelo — APPROVED** [review]

**Pull request merged**